### ISSUE TITLE
Update: specter-desktop 0.5.5

### ISSUE THREAD

**Issue by k9ert** [raspiblitz/raspiblitz#1171]
Specter-desktop created the 0.3.0 release on 11th may. It updates the decpendency of HWI to 1.1.0.
HWI 1.1.0 is dependendent on pyside2>=5.14.0,<6.0.0
https://github.com/bitcoin-core/HWI/blob/master/setup.py#L24

However, pyside2 is not available on the arm-platform. This results in a failure of the installation of specter-desktop. I hope i can find a solution by making that dependency optional.

In the meantime i'll post a workaround in the next comment soon.

**Comment by k9ert** [comment]
After you have installed Cryptoadvance-specter unsuccessfull, leave the raspiblitz-menu or press CTRL-C. On the commandline do this:
```
sudo su - bitcoin
cd .specter
. ./.env/bin/activate
# Install all the dependies of hwi one by one
pip3 install 'ecdsa>=0.13.0,<0.14.0' 'hidapi>=0.7.99,<0.8.0' 'libusb1>=1.7,<2.0' 'mnemonic>=0.18.0,<0.19.0' 'pyaes>=1.6,<2.0' 'typing-extensions>=3.7,<4.0'
pip3 install hwi --no-dependencies
curl https://raw.githubusercontent.com/cryptoadvance/specter-desktop/master/requirements.txt | xargs -n 1 pip install
pip3 install cryptoadvance.specter --no-dependencies
exit
sudo service cryptoadvance-specter restart
```

After that, specter should be available as shown on the menu.

**Comment by pretyflaco** [comment]
I can confirm that this solved it and I can run Specter Desktop as described in Menu

**Comment by rootzoll** [comment]
@k9ert can you make a PR that will fix the Specter Install-Script `/home.admin/config.scripts/bonus.cryptoadvance-specter.sh`? Thx.

**Comment by k9ert** [comment]
I have to fix the issue entirely. What's imho missing in the install-script is that it's not failing but pretending to have suceeded. That's far from gracufully failing.
I could potentially include this workaround in the install-script but that's definitely a workaround, not a solution.
... and it looks good to have a solution soon (which doesn't need any changes in the install-script).

**Comment by rootzoll** [comment]
Yes thats the evolution of this install scripts - first make them work, then make them robust. Looking forward to your update of the script ... lets us know if we can help out.

**Comment by rootzoll** [comment]
updating to Specter 0.4.0
https://twitter.com/k9ert/status/1267880096003653632?s=20

**Comment by rootzoll** [comment]
OK Specter 0.4.0 will now run again with v1.6 release .. please test once we have a RC1 out.
I have fixed the version on install so then pips get new version that it stays working ;)

**Comment by k9ert** [comment]
I'm quite sure that 0.4.0 is not yet fixing that issue. Effectively specter-desktop can't fix this issue. It need to be fixed in [HWI](https://github.com/bitcoin-core/HWI/issues/331). This issue is fixed but we need a new HWI-Release to really fix it for us. As soon as that happened, we'll create a new specter-desktop release to finally fix it.

**Comment by k9ert** [comment]
I wasn't aware that HWI created a new release. We'll create a new release of specter-desktop this afternoon. Should be fixed soon.

**Comment by k9ert** [comment]
We did the release 0.4.1 and i've tested it with the improved installation-script. IMHO we can close this issue. PR is to master. Not sure that was right so.

**Comment by rootzoll** [comment]
@k9ert thanks, installation looking good on my machine - will be part of v1.6 release

**Comment by openoms** [comment]
Will need two corrrections:
- [X] the .onion address uses https, but forwarded to to the port 80. Needs to use the port 443 (need to accept the self signed cert even in the Tor browser) 
- [X] the .onion address does not fit in the status menu. Need to increase the window size

**Comment by rootzoll** [comment]
https://github.com/cryptoadvance/specter-desktop/releases/tag/v0.5.5 is out

**Comment by rootzoll** [comment]
The pip was already updated: https://pypi.org/project/cryptoadvance.specter and the install script installs the latest ... so give it a test run on RC.

**Comment by rootzoll** [comment]
Also check if its working with the nginx https ... because in the install script it still creates its own https

**Comment by rootzoll** [comment]
@frennkie specter is still using its own HTTPS cert ... would it make sense to also put it behind the nginx HTTPS proxy instead?

**Comment by rootzoll** [comment]
OK tested and reinstalls during update. Will be part of v1.6. Closing issue.

**Comment by ben-kaufman** [comment]
Hi, if that's not too late to make it for the v1.6 release, we just released [Specter v0.6.0](https://github.com/cryptoadvance/specter-desktop/releases/tag/v0.6.0), which among other things also includes a fix to a potential installation issue with Specter v0.5.5. If that's not too late already it will be really great to have that included in the coming release.
Another new notable thing in this release is that it also includes binaries you could use to run Specter instead of downloading from pip if you find that preferable.

**Comment by rootzoll** [comment]
@ben-kaufman there will be a service realease v1.6.1 not long after v1.6 there we can include the newest specter.
